LoopKitUI/Extensions/ChartAxisValuesStaticGenerator.swift

```diff
@@ -93,6 +93,71 @@ extension ChartAxisValuesStaticGenerator {
             return []
         }
     }
+    
+    // This is the same as SwiftChart ChartAxisValuesStaticGenerator.generateYAxisValuesWithChartPoints(...) with the exception that the `firstValue` calculation has been corrected
```

Comment:
Maybe describe what was wrong, or changed about `firstValue`? Just saying it was "corrected" isn't very clear.